sim/paladin/exorcism.go

```diff
@@ -51,7 +51,7 @@ func (paladin *Paladin) registerExorcism() {
 			SpellSchool: core.SpellSchoolHoly,
 			DefenseType: core.DefenseTypeMagic,
 			ProcMask:    core.ProcMaskSpellDamage,
-			Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
+			Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL | core.SpellFlagIgnoreResists,//Logs show it never has partial resists, No clue why, still misses
```

Comment:
I think the flag you're looking for here is `SpellFlagBinary`. `SpellFlagIgnoreResists` causes it to not be affected by resistance stats

Reply:
Well there is no Holy Resistance in the game, and IDK what is Binary about the spell, maybe it is flagged as binary? IDK how to look that up. Results are the same so I updated it

Reply:
Oh ok sorry @mlizard32 could you add that flag back per what @wsphillips said?

Reply:
Ya it full resists, which I believe is a "Miss", with either flag it will "Miss". 
Its super weird, on the era logs you can see it has partial resists, its some weird qurik with the SoD version. 
The end result is the same with either flag, I'm not really sure which makes more sense

Reply:
I'm kinda wondering if the "100% crit if demon/undead" somehow needed to be coded in a "Binary" way so thats why it behaves like a "Binary" spell now?

Reply:
Ok are we good then do you think? That was my only concern so I can merge this otherwise

Reply:
K I think It is "Binary" and here is my reasoning.

Binary spells are spells that have additional effects when cast, examples I looked at (pyroblast, shadow flame, frostfire bolt) these spells do not seem to partially resist according to logs I looked at, while all spells I looked at (shadow bolt, Holy Shock, Fire blast) with 1 simple effect do partially resist.

Comparing original Exorcism to new Exorcism, the new Exorcism has an additional "Effect" (a dummyscript).
Since the new spell has an additional "Effect" it is now a binary spell.

https://www.wowhead.com/classic/spell=415073/exorcism
https://www.wowhead.com/classic/spell=10314/exorcism

My vote is to leave the flag as Binary - @kayla-glick had it right.